setup.cfg

```diff
@@ -31,7 +31,7 @@ project_urls =
 [options]
 packages = find:
 install_requires =
-    awkward>=1
+    awkward>=2
```

Comment:
What's the proper way to handle this new version pin?

What I mean is `fastjet<=3.4.0.1` actually only works with `awkward>=1,<2`, but since we already published that, there's no way for pip, etc. to know so effectively all versions of `fastjet<=3.4.0.1` will break if installed with `awkward>=2`.

Or is this not a problem because once we release `fastjet 3.4.0.2`, which will work with `awkward 2.0.0`, everything will be fine?

This is more a meta question not directly related to this PR. But I welcome any guidance.

Reply:
@henryiii has had a lot to say about this: https://iscinumpy.dev/post/bound-version-constraints/

Applying this case to his arguments, fastjet is a library, to be used as a dependency in someone else's (otherwise version-constrained) application, so it should _not_ be version-capped. Also,

> once we release `fastjet 3.4.0.2`, which will work with `awkward 2.0.0`, everything will be fine

is a good argument. There's a short interval of time in which someone attempting to install will get a combination of packages that don't work together, and that's why I wanted to close the gap as quickly as possible. If someone installs in such a way that they deliberately hold `fastjet` back while allowing `awkward` to update, they'll make problems for themselves. (This is, I think, a summary of the argument against version-capping—adding to it the fact that the "problems they make for themselves" might have been made by one of their dependencies!)

I suppose it's also possible that a dependency management system would upgrade `awkward` without also upgrading `fastjet` without having any positive constraints to do so. I'm quite sure that conda does not work this way, and I'm pretty sure that pip doesn't work this way, or at least not anymore.

Reply:
ok great, i was vaguely remembering this post and I'm glad to have been pointed to it again!